src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PerformanceAnalyzerController.java

```diff
@@ -30,7 +31,7 @@
     private boolean rcaEnabled;
     private boolean loggingEnabled;
     private boolean batchMetricsEnabled;
-    private volatile int shardsPerCollection;
+    @VisibleForTesting volatile int shardsPerCollection;
```

Comment:
shardsPerCollection is not used by testing right ?

Reply:
good catch, reverting back